Day2/Day2.py

```diff
@@ -20,6 +20,25 @@ def analyse(segment):
     else:
         return False
 
+def interpret(part):
+    datarange = part[0]
+    letter = part[1][0]
+    password = part[2]
+    occurences = 0
+    firstlast = datarange.split("-")
+    first = int(firstlast[0])
+    last = int(firstlast[1])
+    pass1 = password[first]
+    pass2 = password[last]
```

Comment:
The error you're running into are in these lines, and it's kinda subtle:

Let's say your input is `1-5 a: abcde`. The problem wants you to check the first and fifth element, so in this case you would be running `pass2 = "abcde"[5]`. Can you spot what the issue is with this?

(hint: remember that in python, the first element of a string or array is element 0